server/chart_config.json

```diff
@@ -43,11 +43,23 @@
     "relatedChart": {
       "title": "Labor Force Participation Rate",
       "scale": true,
-      "denominator": "Count_Person",
+      "denominator": "Count_Person_16OrMoreYears",
```

Comment:
@shifucun @beets 

This appears to have worked out (thanks to code elsewhere, likely by luck).

Count_Person_InLaborForce exists both from BLS and from WDI. The overlap is only on country/USA and we seem to pick WDI. Bar charts comparing USA with other countries LG since all use WDI. Bar charts for places in USA almost look good:

![image](https://user-images.githubusercontent.com/18621425/95936065-910a8b00-0d89-11eb-85ba-2640564ec57e.png)


It says "places in" instead of "USA and places in" so it seems ok that USA is missing (likely bc different source?)

Furthermore, our code appears to be smart and only graphs one of the Labor Force Participation Rate charts (the relatedChart config here and the chart config below both have the same title). In this case, the one from WDI:

![image](https://user-images.githubusercontent.com/18621425/95936172-d4fd9000-0d89-11eb-8ff1-c0321fd1ffe7.png)